Library/Homebrew/cmd/update.sh

```diff
@@ -219,8 +219,16 @@ pull() {
 
   if [[ -z "$HOMEBREW_MERGE" ]]
   then
+    if [[ -n "$HOMEBREW_VERBOSE" ]]
+    then
+      echo "Rebasing $DIR onto origin/$UPSTREAM_BRANCH ..."
+    fi
     git rebase "${QUIET_ARGS[@]}" "origin/$UPSTREAM_BRANCH"
   else
+    if [[ -n "$HOMEBREW_VERBOSE" ]]
+    then
+      echo "Merging origin/$UPSTREAM_BRANCH into $DIR..."
```

Comment:
The tiniest nit, but there's a missing space between `$DIR` and `...` which sticks out because it is present in the other message you added.